From ffd3369700758e9e77f057a8f2e5aa2e651c8a16 Mon Sep 17 00:00:00 2001
From: svonlanthen <svonlanthen@snowflake.ch>
Date: Thu, 6 Feb 2014 16:18:22 +0100
Subject: [PATCH] Task NR Comment

---
 index.html | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100644 index.html

diff --git a/index.html b/index.html
new file mode 100644
index 0000000..6d798f7
--- /dev/null
+++ b/index.html
@@ -0,0 +1,3 @@
+hello world
+develop branch test
+change example
\ No newline at end of file
-- 
1.8.5.2.msysgit.0

